django/db/backends/sqlite3/features.py

```diff
@@ -9,6 +9,14 @@
 from .base import Database
 
 
+def supports_cloning():
+    if (int(platform.python_version().startswith('2') or
+            int(platform.python_version_tuple()[1]) < 7)) \
```

Comment:
You are checking for Python versions less than 3.7 here? Something like this should work?

```suggestion
    if sys.version_info < (3, 7)
```

Reply:
This is not a reliable check, Python can be shipped/compiled with different versions of SQLite (see https://github.com/django/django/pull/12646#issuecomment-745138590) You should check SQLite version instead:
```python
can_clone_databases = Database.sqlite_version_info >= (3, 27, 0)
```

Reply:
I've changed the cloning method from ```VACUUM INTO``` which only supports SQLite 3.27+ to ```backup()``` which supports SQLite 3.6.11+, but the catch is that ```backup()``` doesn't support any python 3.6 since it was introduced in 3.7.

Reply:
> ... the catch is that backup() doesn't support any python 3.6 since it was introduced in 3.7.

Happy to see us hit 3.2, but we'll be dropping Python 3.6 and 3.7 from `master` after we branch `stable/3.2.x` — so it might be that we target that (if it makes everything simpler). That means users wouldn't get it in their own projects for another version, but we would have it for development from just as soon now. Thoughts?

Reply:
Looking at the Django survey about 4% of users use exclusively Windows+sqlite, and about 10% of users use this combination some of the time. (Have I understood correctly that we're looking at sqlite and windows users here)

Reply:
Well... `./manage.py test` would benefit across all DBs no? (The issue is can we make it work for that combo... — but also some macOS user who's got version IDK of SQLite — i.e. can we fallback gracefully.) 

My thought is just that maybe (**maybe**) we can opt for the easier path, if that's available after branching 3.2… (First preference is to get it in, but balancing effort/complexity to reward)

Reply:
@carltongibson I'd rather see users get access to it now excluding 3.6 support than waiting for another version or using it experimentally on the dev version.

If we miss the time for balancing it over the next two weeks, we can delay as suggested, but the sooner we get this in the better I believe.

Are there issues running this patch on macOS?